modules/data-table/main/theme.scss

```diff
@@ -51,9 +51,11 @@
     }
   }
 
-  .hx-data-table-footer {
+  .hx-data-table-control-panel-bottom,
+  .hx-data-table-control-panel-compact,
+  .hx-data-table-control-panel {
     background: $footer-background-col;
-    box-shadow: 1px 1px 1px 1px $footer-shadow-col;
+    box-shadow: 1px 1px 1px $footer-shadow-col;
```

Comment:
This was not consistent with the style for the compact rows - It looked nicer to change it this way as the box shadow is slightly smaller